assets/js/atomic/blocks/product-elements/button/edit.js

```diff
@@ -1,33 +1,30 @@
 /**
  * External dependencies
  */
-import { __ } from '@wordpress/i18n';
 import { Disabled } from '@wordpress/components';
 import { useBlockProps } from '@wordpress/block-editor';
+import { useEffect } from 'react';
```

Comment:
Is there any reason why we don't use `useEffect` from `@wordpress/element` here?

_(Came across this PR while exploring the Query Block.)_

Reply:
Absolutely no :'(  
During the cooldown, I guess that it makes sense to work on this https://github.com/woocommerce/woocommerce-blocks/issues/6732

I created a PR and assigned you as a reviewer https://github.com/woocommerce/woocommerce-blocks/pull/7298. Thanks for catching this!